test/Common/AutoFixture/Attributes/RepeatingPatternBitAutoDataAttribute.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Reflection;
+
+namespace Bit.Test.Common.AutoFixture.Attributes;
+
+public class RepeatingPatternBitAutoDataAttribute : BitAutoDataAttribute
+{
+    private readonly List<object[]> _repeatingDataList;
+
+    public RepeatingPatternBitAutoDataAttribute(object[] first)
```

Comment:
```suggestion
    public RepeatingPatternBitAutoDataAttribute(object?[] first)
```
💭  Can we add nullable reference support for the constructors? Would require adding `#nullable enable` to the top of the file.

We want to support nulls in the array:`[RepeatingPatternBitAutoData([null, "", " "])`.

I don't think we want to support `[RepeatingPatternBitAutoData(null))` or `[RepeatingPatternBitAutoData([]))`. The first errors and the later skips the tests. Should we throw our own errors in these cases?

Reply:
Fixed in [d8b2235](https://github.com/bitwarden/server/pull/5167/commits/d8b2235a81c745cb207eeb0c041bd81c566cbe54)